frontend/src/components/NavBar/NavBar.jsx

```diff
@@ -2,10 +2,11 @@ import React from 'react';
 import { Link, useHistory } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
+import { startCase } from 'lodash/fp';
```

Comment:
Same suggestion, when we import from lodash/fp let's just import the util we need not everything.

```suggestion
import startCase from 'lodash/fp/startCase';
```

Reply:
Thanks!